site/content/en/docs/start/_index.md

```diff
@@ -423,15 +423,63 @@ choco install minikube
 {{% /quiz_instruction %}}
 
 {{% quiz_instruction id="/Windows/x86-64/Stable/.exe download" %}}
-Download and run the stand-alone [minikube Windows installer](https://storage.googleapis.com/minikube/releases/latest/minikube-installer.exe).
+1. Download the [latest release](https://storage.googleapis.com/minikube/releases/latest/minikube-installer.exe).  
+<br>
+    Or if you have `curl` installed, use this command:
+    ```shell
+    curl -Lo minikube.exe https://storage.googleapis.com/minikube/releases/latest/minikube-installer.exe
+    ```
 
-_If you used a CLI to perform the installation, you will need to close that CLI and open a new one before proceeding._
+2. Add the binary in to your `PATH`.
+    - Create a new folder in C drive, name it `minikube` (recommended) or whatever you like.
+
+    - Move the `minikube.exe` to that folder.
+    - Click on the Windows start menu. Start typing "environment".
+    - You'll see the search result **Edit the system environment variabless.** Select it.
+    - A System Properties window will popup. Click the **Environment Variables** button at the bottom.
+    - Select the `Path` variable under **System variables**. Click the **Edit** button.
+    - Click the **Add** button and paste in the folder path where `minikube.exe` is placed.
+    - Click **OK** as needed. Close all remaining windows by clicking **OK**.
+    - Reopen Command prompt window, and run.
+
+    _If you used a CLI to perform the installation, you will need to close that CLI and open a new one before proceeding._
 {{% /quiz_instruction %}}
 
 {{% quiz_instruction id="/Windows/x86-64/Beta/.exe download" %}}
-Download and run the stand-alone minikube Windows installer from [the release page](https://github.com/kubernetes/minikube/releases).
-
-_If you used a CLI to perform the installation, you will need to close that CLI and open a new one before proceeding._
+1. Download the <a href="#" id="latest-beta-download-link">latest beta release</a>.  
+<br>
+    Or if you have `curl` installed, use this command:
+    ```shell
+    r=https://api.github.com/repos/kubernetes/minikube/releases
```

Comment:
I am curious have you verified this yourself on a windows machine?

Reply:
Thanks for pointing this out. It took a while to learn the basics of PowerShell but now I have updated the instructions to set path using the command-line, and have tested everything on Windows.